internal/usecase/interactor/common.go

```diff
@@ -46,6 +45,7 @@ func NewContainer(r *repo.Container, g *gateway.Container, config ContainerConfi
 	}
 }
 
+// common will be decrepated. Please use the Usecase function instead.
```

Comment:
```suggestion
// Deprecated: common will be deprecated. Please use the Usecase function instead.
```
using the `Deprecated:` at the beginning of the comment is a standard mentioned in [godocs](https://go.dev/blog/godoc), also it helps the IDE to give a better experience